sktm/db.py

```diff
@@ -524,31 +525,31 @@ def commit_patch(self, patch_id, patch_name, patch_url, baseurl,
 
         """
         # pylint: disable=too-many-arguments
-        logging.debug("commit_patch: pid=%s", patch_id)
-        source_id = self.get_sourceid(baseurl, project_id)
+        logging.debug("__commit_patch: pid=%s", patch_id)
+        source_id = self.__get_sourceid(baseurl, project_id)
         self.cur.execute('INSERT OR REPLACE INTO patch(id, name, url, '
                          'patchsource_id, date) '
                          'VALUES(?,?,?,?,?)',
                          (patch_id, patch_name, patch_url, source_id,
                           patch_date))
         self.conn.commit()
 
-    def commit_series(self, patches):
+    def __commit_series(self, patches):
```

Comment:
The recent changes call this method from outside, so this marking isn't valid anymore and will break things.

Reply:
Sorry, I do not notice it. I will fix it. Thanks

Reply:
I should look your previous patch after resolving the conflict